reframe/core/exceptions.py

```diff
@@ -88,6 +88,10 @@ class StatisticsError(ReframeError):
     """Raised to denote an error in dealing with statistics."""
 
 
+class BuildSystemError(ReframeError):
+    """Raised when a build system is not configured properly."""
+
+
```

Comment:
I should also remove the `CompilationError`. Not valid any more.

Reply:
Actually, I need to keep the `CompilationError` and change it. Before we used to print the output of the compilation output. We still need to treat such errors specially. Perhaps, we could print the path to the compilation output.